frontend/src/theme.ts

```diff
@@ -29,3 +30,15 @@ const theme = createTheme({
 } as unknown as ThemeOptions);
 
 export default theme;
+
+export const AuthFormWrapper = styled(Box)`
+  min-height: calc(100vh - 64px - 50px);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const PaperForm = styled(Paper)`
+  border-radius: 20px;
+  padding: 2rem;
+`;
```

Comment:
theme.ts should contain only general theme variables and constants, not components

you should create folder in src/components like 'design' or etc, that will contain all our custom styled-components

Reply:
Ok, I'll fix it